e2e/tests/caseManagementOrder_test.js

```diff
@@ -3,25 +3,69 @@ const standardDirectionOrder = require('../fixtures/standardDirectionOrder.json'
 const dateFormat = require('dateformat');
 
 let caseId;
+let today;
 
 Feature('Case Management Order Journey');
 
 BeforeSuite(async (I) => {
   caseId = await I.submitNewCaseWithData(standardDirectionOrder);
+  today = new Date();
 });
 
 Scenario('local authority sends agreed CMO to judge', async (I, caseViewPage, uploadCaseManagementOrderEventPage) => {
   await I.navigateToCaseDetailsAs(config.swanseaLocalAuthorityUserOne, caseId);
+  await localAuthoritySendsAgreedCmo(I, caseViewPage, uploadCaseManagementOrderEventPage);
+  I.seeEventSubmissionConfirmation(config.applicationActions.uploadCMO);
+  caseViewPage.selectTab(caseViewPage.tabs.draftOrders);
+  assertDraftCaseManagementOrder(I);
```

Comment:
minor: not big fan of having status as default value in assertDraftCaseManagementOrder
by reading test I should know what is expected